### ISSUE TITLE
add a link to older binaries

### ISSUE THREAD

**Issue by pieper** [mhalle/slicer4-download#3]
@mhalle For now can you add this link so that older binaries are available:

http://slicer.kitware.com/midas3/folder/274

**Comment by mhalle** [comment]
It works from inside BWH?

-M

**Comment by pieper** [comment]
Not currently, but that's a different issue  ; )

**Comment by mhalle** [comment]
I have an "older releases" link under "Stable Releases".  It links to
slicer.kitware.com .  Is there a difference?

--Mike

**Comment by pieper** [comment]
Yes, the slicer.kitware.com link invokes some big server side process so it
is slow to load.  Also Jc said yesterday that he plans to disable that
link.  The '.../folder/274' link is fast and organized by release number.

-Steve

**Comment by mhalle** [comment]
Instead of disabling slicer.Kitware.com, could you just have it point to folder 274? There needs to be an easy way to get to the downloads, and slicer.Kitware.com is the easiest. If that change can be made, then both the link on the downloads page and the "just go to slicer.Kitware. com" verbal directions will be in sync. That ...folder/274 link just seems obscure.

-M

**Comment by pieper** [comment]
Point taken @mhalle, on the other hand I think the most logical verbal directions are 'go to download.slicer.org' (or just slicer.org) and have the internals of the linkage handled there so, as an example, if we decided to host on something other than midas the change is localized and under our control.  Relying on a redirect on the kitware server seems more fragile.

@jcfr can you comment?

**Comment by mhalle** [comment]
I agree that going to download.slicer.org is the right canonical 
location for downloads.  This whole discussion started with the need to 
kitware for old releases.  All I'm suggesting is the place we link to be 
a reasonably simple URL, which happens to be the URL you suggested in 
the first place.

If @jcfr is already making a change to the way slicer.kitware.com is 
treated, making it point the most obvious location in the data tree 
seems a clean solution with no downsides.

If we move away from kitware, we can easily change the link on 
download.slicer.org .

--Mike

**Comment by jcfr** [comment]
> making it point the most obvious location in the data tree
> seems a clean solution with no downsides

The server has been updated to point to the list of Slicer releases by default.

See http://slicer.kitware.com/

For more details, https://github.com/midasplatform/slicerpackages/commit/4942459bcfffc4cb18079a0c70b02ceea6f33a32

**Comment by jcfr** [comment]
> a reasonably simple URL

URL http://slicer.kitware.com/ can now be used.

**Comment by mhalle** [comment]
Thanks!

-M

**Comment by pieper** [comment]
Hi Mike -

I notice that download.slicer.org doesn't have a link to slicer.kitware.com
- can you please add it?  It could be above the Slicer3 link and be called
  something like "Previous Slicer4 downloads".

Thanks,
Steve

**Comment by mhalle** [comment]
Actually, it is there, under stable downloads label (older releases). 

-Mike

**Comment by pieper** [comment]
Totally missed that!  Am I the only one who would miss it?  I certainly
didn't think to look there.

**Comment by jcfr** [comment]
Since 

(1) this repository has been deprecated in favor of https://github.com/Slicer/slicer_download and 
(2) the issue has been effectively addressed in https://github.com/mhalle/slicer4-download_deprecated/commit/f8fda08a261e3efa5d31c2cc758c07e6c6b5a3d1,

.... this issue should then be **closed**.
